automated-comments/go/hamming/strings_split_used.md

```diff
@@ -0,0 +1,2 @@
+The type in Go to hold a single character is `rune`. A string can be transformed into a `[]rune` (slice of runes) via type conversion: `var aRunes = []rune(a)`.
```

Comment:
This is good advice, but I wonder if we can reframe it in a slightly more positive way. "This exercise involves examining a string character by character (in Go characters are referred to as _runes_, by the way). Splitting the string into a slice of strings, one for each rune, is one way to do this, but it's not the most efficient. Go has a data type specifically for storing individual runes: `rune`. Try converting the string to a slice of runes like this: `ar := []rune(a)`."

Reply:
Per the guidelines, I think we should put the _action first_, but right now there isn't any. This automated mentoring comment are three statements without actionable.

Perhaps:

````suggestion
Improve the efficiency by converting the string to a slice of `rune`s like this:

```go
ar := []rune(a)
```

The type in Go to hold a single character is `rune`. Splitting the string into a slice of single character strings and working with them is a lot slower than ...
````

I want to incorporate everything @bitfield suggested butttt I don't feel very articulate right now 😭